gulp/js/lint.js

```diff
@@ -10,17 +10,18 @@ var gulp = require('gulp');
 var taskName = 'js:lint',
 	taskConfig = {
 		src: [
-
-			// './gulp/**/*.js',
-			// './helpers/**/*.js',
-			// './test/**/*.js',
-			// '!./test/**/expected/**/*.js',
 			'./source/assets/js/**/*.js',
 			'./source/modules/**/*.js',
 			'./source/pages/**/*.js',
 			'./source/demo/modules/**/*.js',
-
-			// './source/demo/modules/!**!/!*.jsx',
+			'./source/demo/pages/**/*.js',
+			'!./source/modules/.scaffold/scaffold.js'
+		],
+		watch: [
```

Comment:
@flavaflo, if the task has its own watcher, we should probably remove it as a dependency of the `js` task: https://github.com/unic/estatico/blob/feature/ESTATICO-170-eslint/gulp/js/default.js#L150 (and add it to `build`, instead).
Otherwise, every change to a JS file will trigger the linting twice when using `--skipWebpackWatch`.